pkg/configfilearg/defaultparser.go

```diff
@@ -7,7 +7,7 @@ import (
 
 func MustParse(args []string) []string {
 	parser := &Parser{
-		After:         []string{"server", "agent", "etcd-snapshot"},
+		After:         []string{"server", "agent", "etcd-snapshot", "save", "delete", "list", "prune"},
```

Comment:
I see this getting unwieldy as we add more commands with subcommands. Can we perhaps enhance it to handle subcommands properly? Perhaps something like `"etcd-snapshot:1"` which would indicate that the etcd-snapshot command may have 1 subcommand after it, and if the 1 next argument after it doesn't start with `--` then the args should be appended after that subcommand.

Reply:
Implemented the "command:NUM" idea, unit test updated.